src/redist/targets/GenerateMSIs.targets

```diff
@@ -306,6 +306,9 @@
         <UpgradeCode>$(Templates50InstallerUpgradeCode)</UpgradeCode>
         <DependencyKeyName>NetCore_Templates_5.0</DependencyKeyName>
       </TemplatesMsiComponent>
+    </ItemGroup>
+
+    <ItemGroup Condition="!$(Architecture.StartsWith('arm'))">
```

Comment:
Are we adding these back later? Technically, the 5.0 SDK is supposed to support 5.0 and all current LTS versions. I don't expect us to support running 2.1/3.1 apps, but creating a project should be supported to maintain feature parity in the SDK.

Reply:
I though so too. I asked @wli3 and the only reason these MSIs exist is for the insertion into VS. Given no ARM64 VS, I opted for not creating them.

Reply:
Oh right, the nupkg content is version agnostic. We only produce (x86/x64) MSIs to install into the correct SDK folder for VS. We're good then.

Reply:
(This does not affect templates that get downloaded. And we still embed the 5.0 template MSI on the box)